integrations/battlelog/models/Overview.js

```diff
@@ -0,0 +1,20 @@
+module.exports = class Overview {
+  constructor(overview) {
```

Comment:
SE overview só contem as propriedades que você quer (nada a mais) você pode usar algo como `Object.assign(this, overview);`

Reply:
pior que nao.. cada json desses eh gigantesco... estou pegando soh as properties que irei utilizar.. ai conforme a necessidade vou adicionando no modeo